react-components/src/components/RuleBasedOutputs/RuleBasedOutputsSelector.tsx

```diff
@@ -1,9 +1,9 @@
 /*!
  * Copyright 2023 Cognite AS
  */
-import { useEffect, type ReactElement } from 'react';
+import { useEffect, type ReactElement, useMemo } from 'react';
 
-import { type CogniteCadModel } from '@cognite/reveal';
```

Comment:
My Lint is requesting and annoying me to add the type :D

Reply:
Mine too :sweat_smile: Forgot to run yarn lint